addons/zylann.hterrain/hterrain_detail_layer.gd

```diff
@@ -35,6 +35,7 @@ export(Texture) var texture setget set_texture, get_texture;
 export(float) var view_distance = 100.0 setget set_view_distance, get_view_distance
 export(Shader) var custom_shader setget set_custom_shader, get_custom_shader
 export(float, 0, 10) var density = 4 setget set_density, get_density
+export(int) var billboard_scale = 1 setget set_billboard_scale, get_billboard_scale
```

Comment:
I'd name it `instance_scale` (as it may not necessarily be a billboard), and it should be `float`